pkg/collector/corechecks/cluster/orchestrator/manifest_buffer.go

```diff
@@ -157,8 +161,11 @@ func BufferManifestProcessResult(messages []model.MessageBody, buffer *ManifestB
 func setManifestStats(manifests []interface{}) {
 	// Number of manifests flushed
 	manifestFlushed.Set(int64(len(manifests)))
+	tlmManifestFlushed.Add(float64(len(manifests)), "orchestrator:k8s")
```

Comment:
If I remember correctly, you only need to supply the tag value here
```suggestion
	tlmManifestFlushed.Add(float64(len(manifests)), "k8s")
```